### PR TITLE
Adding more tests to cover labelDAO operations

### PR DESCRIPTION
**Pull request by pubudu538**

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.core/src/test/java/org/wso2/carbon/apimgt/core/dao/impl/ApiDAOImplIT.java [hunk 1]
```diff
@@ -587,4 +579,35 @@ public void testAddAPIWithoutAddingLabels() throws Exception {
...
+
+        labelNames.add(label2.getName());
+        API substituteAPI = new API.APIBuilder(api).labels(labelNames).build();
+        apiDAO.updateAPI(api.getId(), substituteAPI);
+        API apiFromDB = apiDAO.getAPI(api.getId());
+
+        API expectedAPI = SampleTestObjectCreator.copyAPIIgnoringNonEditableFields(api, substituteAPI);
+        Assert.assertNotNull(apiFromDB);
+        Assert.assertTrue(APIUtils.isListsEqualIgnoreOrder(apiFromDB.getLabels(),expectedAPI.getLabels()),
+                TestUtil.printDiff(apiFromDB, expectedAPI));
```
Comment (chamilaadhi) [inline components/apimgt/org.wso2.carbon.apimgt.core/src/test/java/org/wso2/carbon/apimgt/core/dao/impl/ApiDAOImplIT.java]: Format required here

File: components/apimgt/org.wso2.carbon.apimgt.core/src/test/java/org/wso2/carbon/apimgt/core/dao/impl/LabelDAOImplIT.java [hunk 2]
```diff
@@ -116,4 +166,25 @@ public void testDeleteLabel() throws Exception {
...
+        List<String> accessUrls = new ArrayList<>();
+        accessUrls.add("https://updated.public");
+        Label updatedLabel = SampleTestObjectCreator.createLabel("public").accessUrls(accessUrls).build();
+
+        labelDAO.updateLabel(updatedLabel);
+        List<Label> labelsFromDb = labelDAO.getLabels();
+        Assert.assertNotNull(labelsFromDb);
+        Assert.assertEquals(labelsFromDb.size(), 1);
+        Assert.assertTrue(labelsFromDb.get(0).equals(updatedLabel),
+                TestUtil.printDiff(labelsFromDb.get(0), updatedLabel));
```
Comment (chamilaadhi) [inline components/apimgt/org.wso2.carbon.apimgt.core/src/test/java/org/wso2/carbon/apimgt/core/dao/impl/LabelDAOImplIT.java]: Format required here

File: components/apimgt/org.wso2.carbon.apimgt.core/src/test/java/org/wso2/carbon/apimgt/core/dao/impl/ApiDAOImplIT.java (hunk 1 above)
Comment (pubudu538) [inline components/apimgt/org.wso2.carbon.apimgt.core/src/test/java/org/wso2/carbon/apimgt/core/dao/impl/ApiDAOImplIT.java]: This is already formatted.

File: components/apimgt/org.wso2.carbon.apimgt.core/src/test/java/org/wso2/carbon/apimgt/core/dao/impl/LabelDAOImplIT.java (hunk 2 above)
Comment (pubudu538) [inline components/apimgt/org.wso2.carbon.apimgt.core/src/test/java/org/wso2/carbon/apimgt/core/dao/impl/LabelDAOImplIT.java]: This is also formatted.

File: components/apimgt/org.wso2.carbon.apimgt.core/src/test/java/org/wso2/carbon/apimgt/core/dao/impl/ApiDAOImplIT.java (hunk 1 above)
Comment (chamilaadhi) [inline components/apimgt/org.wso2.carbon.apimgt.core/src/test/java/org/wso2/carbon/apimgt/core/dao/impl/ApiDAOImplIT.java]: Oh sorry. Didn't see it as a parameter to the assertTrue() method :)

**Pull request merged**